ui/src/app/shared/requirements/list/requirements.list.component.ts

```diff
@@ -34,23 +34,32 @@ export class RequirementsListComponent extends Table {
         return this._suggestWithWorkerModel;
     }
 
+    get suggestWithOSArch() {
+        return this._suggestWithOSArch;
+    }
+
     @Output() event = new EventEmitter<RequirementEvent>();
     @Output() onChange = new EventEmitter<Requirement[]>();
 
     availableRequirements: Array<string>;
     workerModels: Array<WorkerModel>;
     _suggest: string[] = [];
     _suggestWithWorkerModel: Array<string> = [];
+    _suggestWithOSArch: Array<string> = [];
+
     loading = true;
 
     constructor(private _requirementStore: RequirementStore, private _workerModelService: WorkerModelService) {
         super();
         this.nbElementsByPage = 5;
+    }
+
+    ngOnInit() {
```

Comment:
I think we don't need to put in ngOnInit(), because you don't use values from input inside

Reply:
i don't know, i just found it easier to have to same behavior, code in the list component and in the form component